### PR TITLE
Structure blog roll as more traditional blog

### PR DESCRIPTION
**Pull request by dominic**
Separates blog posts and paper pages into two separate components, as their contents are fairly distinct.

- /blog loads all blog posts in order
  - summary field is used for post preview
  - we can consider adding pagination once there are enough posts
- /blog/:id continues to link to the posts
- blog posts no longer have a dropdown menu, table of contents or whitepaper navigation UI

### CODE REVIEW

**Comment by dominic** [comment]
`/blog`:

Screen Shot 2020-11-03 at 1 56 01 PM

Blog post:

image

File: src/components/markdown/BlogPostPage.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        />
+        <meta property="og:image:width" content={page.image.width.toString()} />
+      </Helmet>
+
+      <h1 className="pageTitle">{page.title}</h1>
+
+      <div className="postAuthor">{page.author}</div>
+      <div className="postDate">{page.date}</div>
+
+      <div dangerouslySetInnerHTML={{ __html: body }} />
```
Comment (blanchardjeremy) [inline src/components/markdown/BlogPostPage.tsx:46]: This is interesting. Is this to circumvent HTML escaping?

File: src/posts/blog/BlogPosts.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { Helmet } from 'react-helmet-async'
+import { Link } from 'react-router-dom'
+
+import { BlogPostMap } from 'posts/post'
+
+export const BlogPosts: React.FC<{ posts: BlogPostMap }> = ({ posts }) => (
```
Comment (blanchardjeremy) [inline src/posts/blog/BlogPosts.tsx:7]: Is there a reason for using `.FC` when when we use `.FunctionComponent` most other places?

**Review by blanchardjeremy — APPROVED** [review]
LGTM except for the couple of comments above. I'll let you merge it in when you say it's ready.

File: src/posts/blog/BlogPosts.tsx (hunk 2 above)
Comment (dominic) [reply src/posts/blog/BlogPosts.tsx:7]: Exact same thing, so ¯\_(ツ)_/¯. One is just shorter.

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```
[the remark this answers is not in the document]
Comment (dominic) [reply .vscode/settings.json]: Ah, indeed not. Updated!

File: src/components/markdown/BlogPostPage.tsx (hunk 1 above)
Comment (dominic) [reply src/components/markdown/BlogPostPage.tsx:46]: Yep. This is the React way to set unescaped HTML. They really make you work for it. This isn't new code – same stuff that was there before, just copied to a new file.

One could also include the `react-markdown-it` component (https://github.com/sunflowerdeath/react-markdown-it) which is just a shortcut for the same line of code: https://github.com/sunflowerdeath/react-markdown-it/blob/master/src/index.js

**Review by blanchardjeremy — APPROVED** [review]
LGTM

**Pull request merged**